core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -347,6 +347,9 @@ func SendEther(db *gorm.DB, chainID *big.Int, from, to common.Address, value ass
 
 func newAttempt(ethClient eth.Client, ks KeyStore, chainID big.Int, etx EthTx, gasPrice *big.Int, gasLimit uint64) (EthTxAttempt, error) {
 	attempt := EthTxAttempt{}
+	if gasPrice == nil {
+		return attempt, errors.New("gas price missing")
```

Comment:
No, this indicates a programming error. Better to leave as a panic and fix the root cause.